test/acceptance/cucumber.js

```diff
@@ -0,0 +1,13 @@
+
+module.exports = {
+  default: {
+    publishQuiet: true,
+    snippetInterface: 'async-await',
+    paths: ['test/acceptance/features/**/*.feature'],
+    backtrace: true,
+    tags: '@complete',
```

Comment:
Why is one test not marked as complete?  What's the value of adding a half completed test?

Reply:
This is purely for dev reasons. will remove